### PR TITLE
added new fonts, styled all page subheadings

### PR DESCRIPTION
**Pull request by wasegree**

### CODE REVIEW

File: app/assets/stylesheets/application.scss [hunk 1]
```diff
@@ -569,12 +584,22 @@ br {
...
+   font-weight: bold;
+}
+
+.pagecard-title {
+   font-family: 'Varela Round', sans-serif;
+}
+
+// uncontrolledcarousel {
+//    height: 100%;
+// }
```
Comment (elysemontano) [inline app/assets/stylesheets/application.scss]: Please remove commented out code

**Review by triciasykes — CHANGES_REQUESTED** [review]
there's a lot of commented out code....

File: app/javascript/components/pages/Home.js [hunk 2]
```diff
@@ -47,22 +48,22 @@ class Home extends Component {
             <UncontrolledCarousel
                 items={[
                     {
-                    altText: 'Slide 1',
-                    caption: 'Slide 1',
+                    // altText: 'Slide 1',
+                    // caption: 'Slide 1',
```
Comment (elysemontano) [inline app/javascript/components/pages/Home.js]: Is this needed?  (This appears multiple times on this page, so please address this for all of those).  Please remove or uncomment

**Review by elysemontano — CHANGES_REQUESTED** [review]
A few small things to remove

**Review by elysemontano — APPROVED** [review]

**Review by triciasykes — APPROVED** [review]

**Pull request merged**